diff --git a/powerline_shell/themes/default.py b/powerline_shell/themes/default.py
index da0c003..0c378ad 100644
--- a/powerline_shell/themes/default.py
+++ b/powerline_shell/themes/default.py
@@ -16,7 +16,7 @@ class DefaultColor(object):
     HOSTNAME_BG = 238
 
     HOME_SPECIAL_DISPLAY = True
-    HOME_BG = 31  # blueish
+    HOME_BG = 90
     HOME_FG = 15  # white
     PATH_BG = 237  # dark grey
     PATH_FG = 250  # light grey
