diff --git a/powerline_shell/themes/default.py b/powerline_shell/themes/default.py
index da0c003..04ed548 100644
--- a/powerline_shell/themes/default.py
+++ b/powerline_shell/themes/default.py
@@ -16,8 +16,8 @@ class DefaultColor(object):
     HOSTNAME_BG = 238
 
     HOME_SPECIAL_DISPLAY = True
-    HOME_BG = 31  # blueish
-    HOME_FG = 15  # white
+    HOME_BG = 35  # a mid-tone green
+    HOME_FG = 00
     PATH_BG = 237  # dark grey
     PATH_FG = 250  # light grey
     CWD_FG = 254  # nearly-white grey
@@ -61,8 +61,8 @@ class DefaultColor(object):
     GIT_STASH_BG = 221
     GIT_STASH_FG = 0
 
-    VIRTUAL_ENV_BG = 35  # a mid-tone green
-    VIRTUAL_ENV_FG = 00
+    VIRTUAL_ENV_BG = 31  # blueish
+    VIRTUAL_ENV_FG = 15  # white
 
     BATTERY_NORMAL_BG = 22
     BATTERY_NORMAL_FG = 7
