diff --git a/defaults.go b/defaults.go
index 1202e9a..069ccda 100644
--- a/defaults.go
+++ b/defaults.go
@@ -142,7 +142,7 @@ var defaults = Config{
 			EscapedBackslash: `\\\\`,
 			EscapedBacktick:  "\\`",
 			EscapedDollar:    `\$`,
-			EvalPromptPrefix: `PS1="`,
+			EvalPromptPrefix: `PS1=$'\n'"`,
 			EvalPromptSuffix: `"`,
 		},
 		"zsh": {
@@ -151,7 +151,7 @@ var defaults = Config{
 			EscapedBackslash:      `\\`,
 			EscapedBacktick:       "\\`",
 			EscapedDollar:         `\$`,
-			EvalPromptPrefix:      `PROMPT="`,
+			EvalPromptPrefix:      `PROMPT=$'\n'"`,
 			EvalPromptSuffix:      `"`,
 			EvalPromptRightPrefix: `RPROMPT="`,
 			EvalPromptRightSuffix: `"`,
@@ -180,7 +180,7 @@ var defaults = Config{
 
 			HomeSpecialDisplay: true,
 			HomeFg:             15,  // white
-			HomeBg:             31,  // blueish
+			HomeBg:             130,
 			AliasFg:            15,  // white
 			AliasBg:            31,  // blueish
 			PathFg:             250, // light grey
